From: Santiago Vila <sanvila@unex.es>
Date: Thu, 30 Oct 2003 01:53:41 +0100
Subject: Define a fallback value for PATH_MAX on GNU/Hurd.

Description: Define a fallback value for PATH_MAX on GNU/Hurd.
Author: Santiago Vila <sanvila@unex.es>
Bug-Debian: https://bugs.debian.org/218335
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
---
 src/hexedit.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/hexedit.h b/src/hexedit.h
index 064db93..d60e89c 100644
--- a/src/hexedit.h
+++ b/src/hexedit.h
@@ -33,6 +33,9 @@
 #ifdef HAVE_LIMITS_H
 #include <limits.h>
 #endif
+#ifndef PATH_MAX
+#define PATH_MAX 4096
+#endif
 
 #ifdef STDC_HEADERS
 #include <stdlib.h>
